main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+
+	"github.com/mailgun/kafka-pixy/Godeps/_workspace/src/github.com/mailgun/log"
+	"github.com/mailgun/kafka-pixy/pixy"
+)
+
+const (
+	defaultBrokers  = "localhost:9092"
+	defaultUnixAddr = "/tmp/kafka-pixy.sock"
+	defaultPIDFile  = "/tmp/kafka-pixy.pid"
+)
+
+var (
+	cfg     pixy.ServiceCfg
+	pidFile string
+)
+
+func init() {
+	flag.StringVar(&cfg.UnixAddr, "unixAddr", defaultUnixAddr,
+		"Unix domain socket address that the HTTP API should listen on")
+	flag.StringVar(&cfg.TCPAddr, "tcpAddr", "",
+		"TCP address that the HTTP API should listen on")
+	b := flag.String("brokers", defaultBrokers, "Comma separated list of brokers")
+	cfg.BrokerAddrs = strings.Split(*b, ",")
+	flag.StringVar(&pidFile, "pidFile", defaultPIDFile, "Path to the PID file")
+}
+
+func main() {
+	log.Init([]*log.LogConfig{&log.LogConfig{Name: "syslog"}})
+
+	flag.Parse()
+
+	if err := writePID(pidFile); err != nil {
+		log.Errorf("Failed to write PID file, cause=(%v)", err)
+		return
```

Comment:
How about `log.Fatalf`? That will also exit the program with a non-zero exit code.